drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -1509,6 +1511,8 @@ local hue = Driver("hue",
       },
       [capabilities.colorControl.ID] = {
         [capabilities.colorControl.commands.setColor.NAME] = set_color_handler,
+        [capabilities.colorControl.commands.setHue.NAME] = set_color_hue_handler,
+        [capabilities.colorControl.commands.setSaturation.NAME] = set_color_saturation_handler,
```

Comment:
should these just be named `set_hue_handler` and `set_color_handler`?

Reply:
Likely can be changed to `set_hue_handler` and `set_saturation_handler` (I am assuming you meant that).

Reply:
i did!